src/chainstate/burn/db/sortdb.rs

```diff
@@ -2828,7 +2827,7 @@ impl SortitionDB {
         SortitionDB::apply_schema_5(&db_tx, epochs_ref)?;
         SortitionDB::apply_schema_6(&db_tx, epochs_ref)?;
         SortitionDB::apply_schema_7(&db_tx, epochs_ref)?;
-        SortitionDB::apply_schema_8(&db_tx, epochs_ref)?;
```

Comment:
@stjepangolemac This change here fixed the tests...I had assumed that the apply_schema_8 should contain the epochs_ref but this broke a bunch of things...I removed it just to make the tests pass but now I am not sure I should have...perhaps these failing tests were indicative of why the stacks node was getting stuck and we need to accommodate this change as part of the hard forks?